test/e2e/Makefile

```diff
@@ -38,4 +38,8 @@ generator:
 runner:
 	go build -o build/runner ./runner
 
-.PHONY: all node docker generator runner 
+lint:
+	@echo "--> Running linter for E2E"
+	@go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run
+
+.PHONY: all node docker generator runner lint
```

Comment:
I know it's not related to the goal of this PR but here we are missing `docker-debug`, `docker-fast`, and `node-fast`.